### PR TITLE
Update release notes for 0.144

### PR DESCRIPTION
**Pull request by haozhun**

### CODE REVIEW

File: presto-docs/src/main/sphinx/release/release-0.144.rst [hunk 1]
```diff
@@ -7,3 +7,10 @@ General Changes
 
 * Fix already exists check when adding a column to be case-insensitive.
 * Fix correctness issue when complex grouping operations have a partitioned source.
+* Discard output stage json from completion event when it is very long. Add
+  ``event.max-output-stage-size`` config to adjust the threshold.
+
+Hive Changes
+------------
+
+* Add support for creating and inserting into bucketed Hive tables.
```
Comment (cberner) [inline presto-docs/src/main/sphinx/release/release-0.144.rst]: Add documentation for this explaining how to use the table property

Comment (haozhun) [inline presto-docs/src/main/sphinx/release/release-0.144.rst]: Looks like we didn't even document partitioned_by. Let me do it all together sometime later.

File: presto-docs/src/main/sphinx/release/release-0.144.rst [hunk 2]
```diff
@@ -6,4 +6,23 @@ General Changes
 ---------------
 
 * Fix already exists check when adding a column to be case-insensitive.
-* Fix correctness issue when complex grouping operations have a partitioned source.
+* Fix correctness issue when complex grouping operations have a partitioned
```
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.144.rst]: No need to wrap here

File: presto-docs/src/main/sphinx/release/release-0.144.rst [hunk 3]
```diff
@@ -6,4 +6,23 @@ General Changes
...
+  source.
+* Fix missing coercion when inserting ``NULL`` values explicitly into a table.
+* Fix regression that the queries fail when aggregation functions present in
+  ``AT TIME ZONE``.
```
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.144.rst]: Don't wrap these either

File: presto-docs/src/main/sphinx/release/release-0.144.rst [hunk 4]
```diff
@@ -6,4 +6,23 @@ General Changes
 ---------------
 
 * Fix already exists check when adding a column to be case-insensitive.
-* Fix correctness issue when complex grouping operations have a partitioned source.
+* Fix correctness issue when complex grouping operations have a partitioned
+  source.
+* Fix missing coercion when inserting ``NULL`` values explicitly into a table.
```
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.144.rst]: Fix missing coercion when using ``INSERT`` with ``NULL`` literals.

File: presto-docs/src/main/sphinx/release/release-0.144.rst [hunk 5]
```diff
@@ -6,4 +6,23 @@ General Changes
...
+* Fix regression that the queries fail when aggregation functions present in
+  ``AT TIME ZONE``.
+* Fix potential memory starvation when a query is run with
+  ``resource_overcommit=true``.
+* Queries run with ``resource_overcommit=true`` may now be killed before
+  they reach ``query.max-memory``, if the cluster is low on memory.
```
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.144.rst]: Remove comma

File: presto-docs/src/main/sphinx/release/release-0.144.rst [hunk 6]
```diff
@@ -6,4 +6,23 @@ General Changes
...
+* Discard output stage json from completion event when it is very long. Add
```
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.144.rst]: Capitalize JSON

Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.144.rst]: This limit can be configured with ``event.max-output-stage-size``.

File: presto-docs/src/main/sphinx/release/release-0.144.rst [hunk 7]
```diff
@@ -6,4 +6,23 @@ General Changes
...
+  ``event.max-output-stage-size`` config to adjust the threshold.
+* Add support for :doc:`/sql/explain-analyze`.
+* Improve performance when processing results in CLI.
```
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.144.rst]: This should be CLI and JDBC

File: presto-docs/src/main/sphinx/release/release-0.144.rst [hunk 8]
```diff
@@ -6,4 +6,23 @@ General Changes
...
+* Improve performance of ``GROUP BY`` queries.
+
+Hive Changes
+------------
+
+* Fix ORC reader to use correct configuration property for ``hive.orc.stream-buffer-size``.
```
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.144.rst]: "to actually use configuration property"

File: presto-docs/src/main/sphinx/release/release-0.144.rst [hunk 9]
```diff
@@ -6,4 +6,23 @@ General Changes
...
+* Add support for creating and inserting into bucketed Hive tables.
```
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.144.rst]: Remove "Hive"

**Comment by electrum** [comment]
Add release note for "queryUri" change

**Pull request merged**